tests/grem.cpp

```diff
@@ -17,8 +17,8 @@ class EncodeGREM : public ::testing::Test
 	}
 	void SetUp() override
 	{
-		memset(encodeBuffer, 0, sizeof(encodeBuffer));
-		auto res = encodeGREMMessage(&grem, encodeBuffer,
+		memset(&encodeBuffer, 0, sizeof(encodeBuffer));
+		auto res = encodeGREMMessage(0, 0x000000F0, &grem, encodeBuffer,
```

Comment:
Is this related to receiverID and messageCounter? Could they be given to descriptive variables instead that are used?